tests/TestCase/Database/Schema/PostgresSchemaTest.php

```diff
@@ -288,7 +288,7 @@ public function testDescribeTable()
                 'precision' => null,
                 'unsigned' => null,
                 'comment' => null,
-                'autoIncrement' => true,
+                'autoIncrement' => false,
```

Comment:
The schema definition for this table does not define this column as a serial, and instead only has `BIGINT PRIMARY KEY` which is not a serial type.